homeworks/denis.yeromenko_Denis-Yeromenko/homework_7/test_fibonachi.js

```diff
@@ -0,0 +1,15 @@
+const { numFibonachi } = require('./fibonachi.js');
+
+describe('Fibonachi', () => {
+    describe('Check the value in a series of numbers', () => {
+        test('First value', () => {
+            expect(numFibonachi(10)[0]).toBe(0);
+        });
+        test('Six value', () => {
+            expect(numFibonachi(10)[5]).toBe(5);
+        });
+        test('Ten value', () => {
+            expect(numFibonachi()[9]).toBe(34);
+        });
+    });
```

Comment:
tests naming should be self describing

also, you need to check all elements of array by looping over them